# Generated for NaoQi method ALVisionVideoInput.getGVMResolution
# Get the resolution of the requested G.V.M.

# Name under which the G.V.M. is known from ALVideoDevice.
string id
---
# { 0 = kQQVGA, 1 = kQVGA, 2 = kVGA }  -1: can't access video source
int32 value


